builder/src/scss/bootstrap-custom.scss

```diff
@@ -1,5 +1,3 @@
-@import "./node_modules/bootstrap/scss/bootstrap";
```

Comment:
I suspected this removal would cause issues and as it turns out, it does:

```
thunderstore-builder-1        | Error in plugin "sass"
thunderstore-builder-1        | Message:
thunderstore-builder-1        |     src/scss/bootstrap-custom.scss
thunderstore-builder-1        | Error: Undefined variable: "$grid-breakpoints".
thunderstore-builder-1        |         on line 2 of src/scss/bootstrap-custom.scss
thunderstore-builder-1        | >> @each $breakpoint in map-keys($grid-breakpoints) {
```

In general I'd recommend not making unnecessary changes to avoid running into issues like this, especially if it's not clear why the original is the way it is.

I'll fix this before merging